docs/codeql/writing-codeql-queries/introduction-to-ql.rst

```diff
@@ -134,7 +130,9 @@ To import the CodeQL library for a specific programming language, type ``import
    where count(f.getAnArg()) > 7
    select f
 
-➤ `See this in the query console on LGTM.com <https://lgtm.com/query/2096810474/>`__. The ``from`` clause defines a variable ``f`` representing a Python function. The ``where`` part limits the functions ``f`` to those with more than 7 arguments. Finally, the ``select`` clause lists these functions.
+➤ The ``from`` clause defines a variable ``f`` representing a Python function. The ``where`` part limits the functions ``f`` to those with more than 7 arguments. Finally, the ``select`` clause lists these functions.
```

Comment:
We originally used this arrow bullet exclusively for links to the LGTM query console. I think it makes sense for the links to the answers in the article, but this should probably be a standard paragraph. Same comment for similar situations below.

Reply:
Agreed, I updated this in b4f7239f54ceac4c3de50ed4c54c75699034c5a2.